app/backend/src/couchers/resources.py

```diff
@@ -0,0 +1,65 @@
+import functools
+import json
+import logging
+from pathlib import Path
+
+logger = logging.getLogger(__name__)
+
+resources_folder = Path(__file__).parent / ".." / ".." / "resources"
+
+
+@functools.cache
```

Comment:
File will cached the filesystem caches anyway if it is frequently read, not sure if we need to cache it in memory as well. It's small though so probably doesn't matter.